content/_data/changelogs/weekly.yml

```diff
@@ -3960,6 +3960,60 @@
     # pull: 3207 too minor (refactoring)
     # pull: 3870 too minor (Jenkinsfile optimization for Java 11)
     # pull: 3868 too minor (grammar)
+- version: '2.164'
+  date: 2019-02-10
+  changes:
+  # pull: 3874 too minor (NodeMonitor message)
```

Comment:
Looks good enough for a backportable bugfix. I would prefer to have it in the changelog